### PR TITLE
Enable ServerTelemetryChannel in .net core by the default

### PR DESCRIPTION
**Pull request by iusafaro**
Enable ServerTelemetryChannel in .net core by the default

### CODE REVIEW

File: test/MVCFramework20.FunctionalTests/FunctionalTest/TelemetryModuleWorkingMvcTests.cs [hunk 1]
```diff
@@ -17,7 +18,18 @@ public TelemetryModuleWorkingMvcTests(ITestOutputHelper output) : base(output)
         [Fact]
         public void TestBasicDependencyPropertiesAfterRequestingBasicPage()
         {
-            this.ValidateBasicDependency(assemblyName, "/Home/About/5", InProcessServer.UseApplicationInsights);
+            const string RequestPath = "/Home/About/5";
+
+            using (var server = new InProcessServer(assemblyName))
```
Comment (cijothomas) [inline test/MVCFramework20.FunctionalTests/FunctionalTest/TelemetryModuleWorkingMvcTests.cs]: Is there a reason of moving this line (using (var server = new InProcessServer(assemblyName))) from base class to each of the subclasses?

File: test/FunctionalTestUtils20/InProcessServer.cs [hunk 2]
```diff
@@ -18,22 +16,28 @@ public class InProcessServer : IDisposable
...
 
         public InProcessServer(string assemblyName, Func<IWebHostBuilder, IWebHostBuilder> configureHost = null)
         {
             this.configureHost = configureHost;
+
             var machineName = Environment.GetEnvironmentVariable("COMPUTERNAME");
             this.url = "http://" + machineName + ":" + random.Next(5000, 14000).ToString();
-            this.backChannel = this.Start(assemblyName);
+
+            this.listener = new TelemetryHttpListenerObservable("http://localhost:4001/v2/track/");
```
Comment (cijothomas) [inline test/FunctionalTestUtils20/InProcessServer.cs]: Lets make this a constant.

**Comment by cijothomas** [comment]
Due to some reason build is not triggered automatically! Can you trigger one against this PR. ?

File: test/MVCFramework20.FunctionalTests/FunctionalTest/TelemetryModuleWorkingMvcTests.cs (hunk 1 above)
Comment (iusafaro) [reply test/MVCFramework20.FunctionalTests/FunctionalTest/TelemetryModuleWorkingMvcTests.cs]: It was in base class only for dependencies, but for requests and exceptions, it was out of it in individual cases - i moved it out of here for dependencies as well to be more consistent in all methods.

**Comment by KEMBL** [comment]
Hello guys, 

thank you for enabling of ServerTelemetryChannel in NETCOREAPP2_0, but how to access it now?

in non-Core it is possible like this:

		var channel = new Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.ServerTelemetryChannel();
		channel.StorageFolder = "./CacheFolder/";
		channel.Initialize(TelemetryConfiguration.Active);


how to get the same in core?

**Comment by cijothomas** [comment]
@KEMBL 
Can you open an GH issue, so that the question gets monitored.

Can you retrieve channel like this hostingEngine.Services.GetService<ITelemetryChannel>()) ?

**Pull request merged**